### PR TITLE
Reverse The Text

### PR DESCRIPTION
**Pull request by memoyil**
Reverses The Text

### CODE REVIEW

**Comment by 0crat** [comment]
@memoyil I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/test/java/org/cactoos/text/ReversedTextTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import org.cactoos.TextHasString;
+import org.hamcrest.MatcherAssert;
+import org.junit.Test;
+
+/**
+ * Test case for {@link ReplacedText}.
```
Comment (yegor256) [inline src/test/java/org/cactoos/text/ReversedTextTest.java]: @memoyil typo?

**Comment by yegor256** [comment]
@memoyil one typo

File: src/test/java/org/cactoos/text/ReversedTextTest.java (hunk 1 above)
Comment (memoyil) [inline src/test/java/org/cactoos/text/ReversedTextTest.java]: Sorry for that mistake i corrected it

**Comment by yegor256** [comment]
@rultor try to merge

**Pull request merged**